python/ray/train/lightning/lightning_trainer.py

```diff
@@ -399,6 +407,8 @@ def __init__(
         resume_from_checkpoint: Optional[Checkpoint] = None,
         metadata: Optional[Dict[str, Any]] = None,
     ):
+
+        print("WHAT IS THIS")
```

Comment:
Remove this line?

The lines below is to construct an empty `RunConfig` and `LightningConfig` object if the users didn't provide one.

Reply:
Hahaha oops I was debugging deprecation messages.